db/migrate/20210818115855_create_tickets.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreateTickets < ActiveRecord::Migration[6.1]
+  def change
+    create_table :tickets do |t|
+      t.string :type, null: false
+      t.float :price, null: false
+      t.references :screenings, null: false
```

Comment:
```suggestion
      t.references :screening, null: false
```

Have you ever run the rails console and checked the connection between models?